config/nrfconnect/chip-module/Kconfig

```
@@ -70,3 +70,145 @@ config CHIP_DEBUG_SYMBOLS
 	default y
 	help
 	  Build the application with debug symbols.
+
+# Factory data
+config CHIP_FACTORY_DATA_ENABLE
+	bool "Enables Factory Data support"
+	default n
+	help
+		Enables generating a factory data after build.
+		To generate factory data the nrfconnect_generate_factory_data function from
+		generate_factory_data.cmake should be called.
+		In result a new output files will be created:
+		- merged_fd.hex file containing firmware and factory data merged
+		- factory_data.hex file containing only a factory data with proper memory offset.
+  
+config CHIP_FACTORY_DATA_PARTITION
+	hex "Localization in NVS storage where Factory Data partition will be stored"
```

How about?
```suggestion
	hex "Address in flash memory where Factory Data partition will be stored"
```
NVS is also a name of module in Zephyr that we actually don't use for factory data, so it could be confusing for someone.